db/db.pl

```diff
@@ -2053,9 +2165,13 @@ sub dataNodes
         esDelete("/${PREFIX}tags", 1);
 
         checkForOld5Indices();
-    } elsif ($main::versionNumber <= 51) {
+    } elsif ($main::versionNumber < 52) {
+        createNewAliasesFromOld("users", "users_v6", "users_v5", \&usersCreate);
         sessions2Update();
         checkForOld5Indices();
+        fieldsUpdate();
```

Comment:
You need a huntsCreate() here.   Please test by running `./tests.pl -viewer` on master and then run `/db.pl http://localhost:9200 upgrade` and see if hunts work right.

Reply:
I think it also needs usersUpdate() because of the addition of the packetSearch flag

Reply:
the createNewAliasesFromOld("users", "users_v6", "users_v5", \&usersCreate); should take care of the users table